### ISSUE TITLE
Update composer installation instructions to reference `wp-cli/wp-cli-bundle`.

### ISSUE THREAD

**Issue by TimothyBJacobs** [wp-cli/handbook#271]
The current docs recommend installing composer globally like so:

```sh
composer global require wp-cli/wp-cli
```

It seems like that should be replaced with

```sh
composer global require wp-cli/wp-cli-bundle
```

https://make.wordpress.org/cli/handbook/installing/#installing-via-composer

**Comment by kimdcottrell** [comment]
I've gone ahead and edited this section of the documentation - https://github.com/wp-cli/handbook/pull/279. 

Composer was tested on a PHP 7.2 Docker container with debian-slim as the underlying OS. Windows testing still needs to happen.

**Comment by schlessera** [comment]
Fixed via #279
